client/components/Layout/FormEditScholarship.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { Button, FormControl, Grid, Stack, TextField} from '@mui/material'
+
+function FormEditScholarship(){
+    return(
+        <Stack>
+            <FormControl component="form" sx={{ width: '100%' }}>
+                <TextField id="outlined-basic" label="Scholarship Name" variant="outlined" sx={{ padding: '0px 0px 20px 0px' }}/>
```

Comment:
you can use TextFieldComponent from utils/formComponentUtils to make TextField consistancy across the website and reduce duplicate code